test/models/concerns/pxe_loader_support_test.rb

```diff
@@ -89,14 +89,19 @@ def setup
 
     test "is PXEGrub2 for associated templates" do
       @subject.expects(:os_default_templates).returns([@template_pxelinux, @template_pxegrub, @template_pxegrub2])
-      assert_equal "Grub2 UEFI", @subject.preferred_loader
+      assert_equal "PXELinux BIOS", @subject.preferred_loader
```

Comment:
Minor thing, but I think you have to change the name of the test. Otherwise we have 2 methods with the same name, so only the 2nd test is running.

Reply:
Amended!